<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personal_details</class>
 <widget class="QWidget" name="personal_details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tomato</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/photos/tomato.png</normaloff>:/photos/tomato.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>获得勋章数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>38</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;仿宋&quot;;
</string>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
  </widget>
  <widget class="QLabel" name="clock_num">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="work_num">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tomato_num">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>番茄数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>总使用时长（min）：</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
